src/test/java/redis/clients/jedis/tests/JedisTest.java

```diff
@@ -27,8 +27,8 @@
 public class JedisTest extends JedisCommandTestBase {
   @Test
   public void useWithoutConnecting() {
-    Jedis jedis = new Jedis("localhost");
     jedis.auth("foobared");
+    Jedis jedis = new Jedis("localhost");
```

Comment:
I think this is by accident. Could you revert it, or could you explain why this change is necessary?